src/Microsoft.DotNet.Build.Tasks/PackageFiles/crossgen.sh

```diff
@@ -54,30 +49,29 @@ crossgen_single()
 {
     __file=$1
     if [[ $__file != *.ni.dll && $__file != *.ni.exe ]]; then
-        if [[ ($__file == *.dll && -e ${__file/.dll/.ni.dll}) || ($__file == *.exe && -e ${__file/.exe/.ni.exe}) ]]; then
```

Comment:
No, I will add it back (the removal of if is an artifact of how I developed this change where I started from what was in the CoreFX version of build tools, instead of what was in HEAD).